ts/package.json

```diff
@@ -2,19 +2,22 @@
   "name": "@project-serum/anchor",
   "version": "0.5.1-beta.2",
   "description": "Anchor client",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
+  "main": "dist/index.cjs.js",
```

Comment:
I renamed these all to be in the same directory because microbundle seemed to ignore subdirectories, but I haven't spent much time looking into this yet